build/ci/pr/Jenkinsfile

```diff
@@ -58,3 +68,14 @@ pipeline {
     }
 
 }
+
+def notOnlyDocs() {
+    sh 'git diff --name-status HEAD~1 HEAD > $WORKSPACE/changes.txt'
+    File file = new File(env.WORKSPACE + "/changes.txt")
+    def lines = file.readLines()
+    def counter = 0
+    for (line in lines) {
+        if (line.contains("docs/")) counter++
+    }
+    return counter != lines.size()
```

Comment:
Here is a shorter version that does not write file on the filesystem but that is maybe a little less readable:
```groovy
def notOnlyDocs() {
        // grep succeeds if there is at least one line without docs/
	return sh (
	    script: "git diff --name-status HEAD~1 HEAD | grep -v docs/",
	    returnStatus: true
	) == 0
}
```

Reply:
Thanks! Fixed.
It will also helps because our Jenkins settings doesn't allow usage of `java.io.File` in pipelines